lib/perl/Genome/InstrumentData/Command/MergeAlignments.pm

```diff
@@ -108,6 +113,7 @@ sub params_for_merged_alignment {
         duplication_handler_params => $self->duplication_handler_params,
         duplication_handler_version => $self->duplication_handler_version,
         samtools_version => $self->samtools_version || undef,
+        bedtools_version => $self->bedtools_version || undef,
```

Comment:
I did it because the `samtools_version` did it. :smile: 

This normalizes `0` to `undef`, though that doesn't seem like something we really need to do.  (SoftwareResults already normalize `''` to `undef`, I believe.)  It's probably not useful here.